sorc/fre-nctools.fd/tools/fregrid/bilinear_interp.c

```diff
@@ -1,3 +1,4 @@
+/** @file */
```

Comment:
Can you add even one sentence describing the contents of each file?

Reply:
Done.